### PR TITLE
Add utility script to deploy Stable/CI release version of Kubernetes.

### PR DESCRIPTION
**Pull request by kishen-v**
Changes:
Added script to deploy a Kubernetes cluster using the existing playbook, to make changes in extra-vars-k8s.json and hosts.yml, which are passed as arguments to the ansible-playbook. Renamed extra-vars-k8s-stable.json to extra-vars-k8s.json. 

Motive: 
The existing usage involved retrieving the version from the respective release buckets, and modifications under hosts and extra-vars-k8s files. 

The k8s-installer.sh under hack,  now accepts the IP of the control-plane and the worker nodes and a few additional parameters. The script makes the required changes in the hosts.yml and extra-vars-k8s.json files and deploys the k8s cluster either alpha or stable release based on the flag passed to script (-a and -r, respectively.)

```
[root@test-launcher1 hack]# ./run.sh -w 163.68.64.109 -c 163.68.64.107 -a
163.68.64.109 is reachable
163.68.64.107 is reachable
Fetching latest Alpha CI version..
Alpha release version to be used for cluster deployment: v1.26.0-alpha.1.220+02109414e8816c
No playbook has been passed, defaulting to install-k8s.yml
Ansible Playbook   : install-k8s.yml
Control Node IP    : 163.68.64.107
Worker Node IP(s)  : 163.68.64.109
Approval flag was not set, proceed to run playbook? [y/n]: y
 
PLAY [Install Runtime and Kubernetes] *****************************************************************************************
 
Post installation:
 
[root@test-launcher1 hack]# ssh root@163.68.64.107
Activate the web console with: systemctl enable --now cockpit.socket
 
Last login: Mon Oct  3 10:47:20 2022 from 129.41.86.2
[root@test-debug-m ~]# kubectl get nodes
NAME            STATUS   ROLES           AGE   VERSION
test-debug-m    Ready    control-plane   86s   v1.26.0-alpha.1.220+02109414e8816c
test-debug-w1   Ready    <none>          55s   v1.26.0-alpha.1.220+02109414e8816c
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,29 @@
...
+```shell
+./k8s-installer.sh -w X.X.X.X -c Y.Y.Y.Y -a -y
+```
+To deploy latest Stable release of k8s:
+```shell
+./k8s-installer.sh -w X.X.X.X -c Y.Y.Y.Y -r -y
+```
+To deploy latest Stable release of k8s for perf-tests:
+```shell
+./k8s-installer.sh -p install-k8s-perf.yml -w X.X.X.X -c Y.Y.Y.Y -r -y
```
Comment (mkumatag) [inline README.md:28]: these scripts can refer `./hack/k8s-installer.sh` because readme is in rootdir

**Review by mkumatag — CHANGES_REQUESTED** [review]
minor comment

File: README.md (hunk 1 above)
Comment (kishen-v) [reply README.md]: Updated Readme to point to the script's path as suggested.

**Pull request merged**